libs/designsystem/src/lib/components/radio/radio-group/radio-group.component.ts

```diff
@@ -141,6 +141,7 @@ export class RadioGroupComponent implements AfterContentInit, ControlValueAccess
 
   setDisabledState(isDisabled: boolean) {
     this.disabled = isDisabled;
+    this.changeDetectionRef.markForCheck();
```

Comment:
Is this needed? Doesn't Angular handle that already?

Reply:
Not 100% sure, but actually yes, it is needed. I'm guessing it's needed because of the change detection strategy. I have breaking tests which can't/won't pass, unless it's there as well as some flaky results when trying it out in the cookbook.